eZ/Bundle/EzPublishCoreBundle/Session/Handler/NativeSessionHandler.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+/**
+ * File containing the SymfonyNativeSessionHandler class.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace eZ\Bundle\EzPublishCoreBundle\Session\Handler;
+
+class NativeSessionHandler extends \SessionHandler
+{
+    /**
+     * @param string $savePath      Path of directory to save session files
+     *                              Default null will leave setting as defined by PHP.
+     *                              '/path', 'host:port'
+     * @param string $saveHandler   Could be any handler supported by php
+     *                              Default null will leave setting as defined by PHP.
+     *                              'redis', 'file'
+     *
+     * @see http://php.net/session.configuration.php#ini.session.save-path for further details.
```

Comment:
This is incorrect URL. It should be http://php.net/manual/en/session.configuration.php#ini.session.save-path :wink:

Reply:
good catch !

Fixed